hw11.py

```diff
@@ -0,0 +1,48 @@
+### decorated function ###
+from datetime import datetime
+def decorator(my_func):
+    def deco_func():
+        print(my_func.__name__)
+        print(datetime.now())
+        my_func()

```

Comment:
Use `return` statement here to be able to decorate functions that return some value